### PR TITLE
Remove memcached

### LINKED ISSUE #144: Remove memcached
Since https://github.com/coopdevs/timeoverflow/pull/517, we no longer need Memcached in our infrastructure :tada: !

### PR DESCRIPTION
**Pull request by mperezv**
I modified the cacheserver role to uninstall memcached when executed, also removed memcached from the list of autoupdate packages.

Moreover, I realized that my ssh key had a little mistake and deleted it.

Now, this role removes memcached when it's installed and don't install it if it's not.

Fix #144

### CODE REVIEW

File: pub_keys/marisa.pub [hunk 1]
```diff
@@ -1 +1 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCReyc0uxOCcjI3fdaBim4AIvxe0Xsw65tvATO3oF0ztTu/zSTZrEHIRkhcBTvLbFhDLC6VUI22Z7r2BndzcEWBFw6yLIbBzYpaDULl8Vfh4jUF/y+Vt0MnDGfS6kspyCnNExzfsmPZuswTFonqeN64/YjSmZubVTfa6/Mwzsh0rlj1JXWRY/UQ83D5j9Bq/wB3QmspGQ2d6atFhOoDtYIPL8OhXlkwHKamkQP20rViTI9/8b29Y8ur0rCnwvSO4U9zmwZANeKMMx+HNQXK6Txqhl6Ybfr66CMGfdrJFqTzubpZmnsixrAx+Bzo3qfVY5spit9MpHuBW124SqsgH8pL marisa@ThinkPad-E480
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCReyc0uxOCcjI3fdaBim4AIvxe0Xsw65tvATO3oF0ztTu/zSTZrEHIRkhcBTvLbFhDLC6VUI22Z7r2BndzcEWBFw6yLIbBzYpaDULl8Vfh4jUF/y+Vt0MnDGfS6kspyCnNExzfsmPZuswTFonqeN64/YjSmZubVTfa6/Mwzsh0rlj1JXWRY/UQ83D5j9Bq/wB3QmspGQ2d6atFhOoDtYIPL8OhXlkwHKamkQP20rViTI9/8b29Y8ur0rCnwvSO4U9zmwZANeKMMx+HNQXK6Txqhl6Ybfr66CMGfdrJFqTzubpZmnsixrAx+Bzo3qfVY5spit9MpHuBW124SqsgH8pL marisa
```
Comment (sauloperez) [inline pub_keys/marisa.pub]: why did you need this? The public key seems good as it is.

Comment (mperezv) [reply pub_keys/marisa.pub]: I thought it would be more homogeneous with the other keys this way and that the username would not affect to the key itself. We can don't do it.

What risks are you thinking on?

Comment (sauloperez) [reply pub_keys/marisa.pub]: No idea if there's any risk or not. If your key had that email in there, I guess it's fine. However, if you choose to remove that part, I'd remove it all, including your name.

@enricostano you might know a bit more about pub key contents than me.

Comment (danypr92) [reply pub_keys/marisa.pub]: In any case, we can move out of this PR this change and do it in a separate PR, no @mperezv?

Comment (enricostano) [reply pub_keys/marisa.pub]: I'm OK with the change, probably better if done in another PR.

The only thing that's important to me is that at least the name `marisa` is left there. It's quite useful while auditing servers. It should be our standard.

**Review by enricostano — APPROVED** [review]

File: pub_keys/marisa.pub (hunk 1 above)
Comment (sauloperez) [reply pub_keys/marisa.pub]: agree with @enricostano 's suggestion. Would you mind adding the SSH key in a separate PR before we merge? better to have a purpose-specific PR for each thing.

Comment (mperezv) [reply pub_keys/marisa.pub]: On it! Give me a few minutes to remember how to cherry-pick and I'll do de PR :smiley:

**Comment by sauloperez** [comment]
Cool @mperezv !

**Review by sauloperez — APPROVED** [review]
